src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -49,6 +53,42 @@ public abstract class AzureServiceBusMessagePump<TMessage> : MessagePump<TMessag
         /// </summary>
         protected string Namespace { get; private set; }
 
+        /// <summary>
+        /// Gets the unique identifier for this background job to distinguish this job instance in a multi-instance deployment.
+        /// </summary>
+        public string JobId { get; }
+
+        /// <summary>
+        /// Triggered when the application host is ready to start the service.
+        /// </summary>
+        /// <param name="cancellationToken">Indicates that the start process has been aborted.</param>
+        public override async Task StartAsync(CancellationToken cancellationToken)
+        {
+            if (Settings.Options.IncludeTopicSubscription)
+            {
+                ServiceBusConnectionStringBuilder serviceBusConnectionString = await GetServiceBusConnectionStringAsync();
+
+                Logger.LogTrace("[Job: {JobId}] Creating subscription '{SubscriptionName}' on topic '{TopicPath}'...", JobId, _subscriptionName, serviceBusConnectionString.EntityPath);
+                var subscriptionDescription = new SubscriptionDescription(serviceBusConnectionString.EntityPath, _subscriptionName)
+                {
+                    AutoDeleteOnIdle = TimeSpan.FromHours(1),
```

Comment:
Let's not set this one so that ops can troubleshoot things

Reply:
Yes.